api/src/http.rs

```diff
@@ -488,6 +488,8 @@ pub enum ApiRequest {
     GetBlobObject(BlobCacheObjectId),
     /// Delete a blob cache entry
     DeleteBlobObject(BlobCacheObjectId),
+    /// Delete a blob cache file
+    DeleteBlobFile(String),
```

Comment:
Looks like the new blob APIs(including this one) are not documented in `api/openapi/nydus-api-v2.yaml` which still says `blob_objects`. Please update the openapi doc as well. Ideally, we should have a CI to validate the openapi interface...